pkgs/development/python-modules/pick-colour-picker/default.nix

```
@@ -0,0 +1,59 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, pygobject3
+, pycairo
+, glib
+, gtk3
+, gobject-introspection
+, wrapGAppsHook
+}:
+
+buildPythonPackage rec {
+  pname = "pick-colour-picker";
+  version = "1.5.0-3ec940";
+
+  src = fetchFromGitHub {
+    owner = "stuartlangridge";
+    repo = "ColourPicker";
+    rev = "3ec9406d787ce373f6db0d520ed38a921edb9473";
+    sha256 = "04l8ch9297nhkgcmyhsbg0il424c8vy0isns1c7aypn0zp0dc4zd";
+  };
+
+  propagatedBuildInputs = [
+    glib
+    pygobject3
+    pycairo
+    gtk3
+  ];
+
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook
+  ];
+
+  strictDeps = false;
+
+  preDistPhases = "fixupIconPath";
+
+  fixupIconPath = ''
+    pickLoc=$(echo "$out/lib/python"*"/site-packages/pick")
```

```suggestion
    pickLoc="$out/${python.sitePackages}/pick"
```

Though even better would be figuring out how does snap do this. I doubt they use hacks like this.

---

Actually this is because of https://github.com/NixOS/nixpkgs/issues/23438